product_kit_management/models/sale_order_line.py

```diff
@@ -0,0 +1,73 @@
+from odoo import Command, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    is_kit = fields.Boolean(string="Is Kit", related="product_id.is_kit", store=True)
+    kit_component_ids = fields.One2many(
+        'product.kit.component', 'parent_order_line_id',
+        compute="_compute_sub_products", store=True, string="Kit Components",
+        help="List of sub products"
+    )
+    is_kit_component = fields.Boolean(string="Is Kit Component?", default=False)
```

Comment:
```suggestion
    is_kit_component = fields.Boolean(string="Is Kit Component?",)
```
do we need to define it?

Reply:
No, by default it will be false only. will remove it.